rules:
- id: unchecked-subprocess-call
  patterns:
  - pattern-not-inside: $S = subprocess.call(...)
  - pattern-not-inside: subprocess.call(...) == $X
  - pattern: subprocess.call(...)
  message: >-
    This is not checking the return value of this subprocess call; if it fails no exception will be raised. Consider
    subprocess.check_call() instead
  languages:
  - python
  severity: WARNING
  fix: subprocess.check_call(...)
